src/parser/FStar.Parser.Dep.fs

```diff
@@ -1664,8 +1724,12 @@ let print_full (deps:deps) : unit =
     in
     let print_all tag files =
         pr tag;
-        pr "=\\\n\t";
-        List.iter (fun f -> pr (norm_path f); pr " \\\n\t") files;
+        if ninja then pr " = " else pr "=\\\n\t";
```

Comment:
or maybe a helper `print_separator` that would be the only place that branches on `ninja`?